src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -285,11 +285,48 @@ private DecisionDagBuilder(CSharpCompilation compilation, LabelSymbol defaultLab
                 case BoundRecursivePattern recursive:
                     MakeTestsAndBindings(input, recursive, tests, bindings);
                     break;
+                case BoundITuplePattern iTuple:
+                    MakeTestsAndBindings(input, iTuple, tests, bindings);
+                    break;
                 default:
                     throw new NotImplementedException(pattern.Kind.ToString());
             }
         }
 
+        private void MakeTestsAndBindings(
+            BoundDagTemp input,
+            BoundITuplePattern pattern,
+            ArrayBuilder<BoundDagTest> tests,
+            ArrayBuilder<BoundPatternBinding> bindings)
+        {
+            var syntax = pattern.Syntax;
+            var patternLength = pattern.Deconstruction.Length;
```

Comment:
>var [](start = 12, length = 3)

nit: `int`

Reply:
Isn't it obvious that a length is an int? Why is this line different than all the others?

---
In reply to: [206292578](https://github.com/dotnet/roslyn/pull/28859#discussion_r206292578) [](ancestors = 206292578)

Reply:
It's a nit, but I don't see a good reason to use `var` for type names that are so short (like `int` or `bool`). For long type names I can see some benefits ;-)

Reply:
I see them as distracting unless they convey something relevant to the reader.